M2/Macaulay2/packages/Complexes.m2

```diff
@@ -117,11 +115,22 @@ directSum Sequence := args -> (
         if y =!= null then y.cache#key = S;
         S))
 
+Hom(Module, Module) := Module => (M,N) -> (
+    Y := youngest(M.cache.cache,N.cache.cache);
+    if Y#?(Hom,M,N) then return Y#(Hom,M,N);
+    H := trim kernel (transpose presentation M ** N);
+    H.cache.homomorphism = (f) -> map(N,M,adjoint'(f,M,N), Degree => first degrees source f + degree f);
```

Comment:
Mike, please just fix this in the core.

Reply:
> Mike, please just fix this in the core.

Oops, good point!  I forgot that that got fixed/cahnged there.  There might be others too.  I'll take a look.